events/code_review_events/bugbug_utils.py

```diff
@@ -199,6 +199,7 @@ def should_run_test_selection(self, build):
                 {
                     "PHABRICATOR_DEPLOYMENT": self.phabricator_deployment,
                     "DIFF_ID": build.diff_id,
+                    "RUNNABLE_JOBS": f"https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.try.revision.{revision}.firefox.decision/artifacts/public/runnable-jobs.json",
```

Comment:
I would prefer to use the configured taskcluster buildUrl function `taskcluster_config.get_service('index').buildUrl('findArtifactFromTask', 'gecko.v2.try.revision.{revision}.firefox.decision', 'public/runnable-jobs.json')`

Reply:
Good idea, done with 29bf899.

I've also added a test.